proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -114,6 +116,49 @@ service PluginCurationService {
   rpc GetLatestCuratedPlugin(GetLatestCuratedPluginRequest) returns (GetLatestCuratedPluginResponse);
 }
 
+// PluginCodeGenerationService generates code using remote plugins.
+service PluginCodeGenerationService {
+  // GenerateCode generates code using the specified remote plugins.
+  rpc GenerateCode(GenerateCodeRequest) returns (GenerateCodeResponse);
```

Comment:
Open to naming here - I preferred this over something like `GeneratePlugins` because it more accurately refers to what the RPC does.

Reply:
`GenerationService`
`CodeGenerationService`

Are a bit shorter and still reflect what the action does .. "generating code". I dunno, just a thought.

Reply:
I like it - renamed to `CodeGenerationService`.